### ISSUE TITLE
[FR] Quarterly statistics

### ISSUE THREAD

**Issue by Dina-desoki** [Razeeman/Android-SimpleTimeTracker#259]
Hi
Would it be possible to add quarterly statistics. I find that the gap between monthly and yearly statistics is huge.
Thanks.

**Comment by Razeeman** [comment]
Hello! It is possible, but there are already a lot of time ranges, will have to think about how to incorporate it without overloading the interface.

**Comment by Dina-desoki** [comment]
The problem with using custom time ranges is that I can not directly compare time ranges. For example, when choosing the "month time range", I have the option to see the previous month (months). But once I choose a custom time range, I lose this feature. If I wish to see statistics for 2024 in quarters, I have to manually change the time range 4 times, one time for each quarter.

**Comment by Cancoillotte** [comment]
Indeed, I'm making the same observations.  Why shouldn't it be possible, once a custom time interval has been selected, to see the same interval before or after it, and so on?

And it wouldn't make the interface any heavier.

**Comment by Razeeman** [comment]
Selecting the same custom interval before or after current one - would be possible to implement. It wouldn't be exactly "quarterly" thought, because months have different number of days, so selecting custom range for example "jan 01 - march 31" and shifting it forward would be something like "april 01 - june 28", hope this makes sense. If this is ok, then it would be the easiest way probably. 

Another idea would be a more "custom" way to select custom range, for example an option to select months instead of the days. Then one could select a custom range "jan - march" and shifting it forward would be "april - june" which would be more "quarterly".

**Comment by Cancoillotte** [comment]
Thank you so much for your attention to each comment!

Both proposals would be useful, in my opinion, but the simpler one would already be an appreciable improvement.

**Comment by timkrief** [comment]
> Indeed, I'm making the same observations. Why shouldn't it be possible, once a custom time interval has been selected, to see the same interval before or after it, and so on?
> 
> And it wouldn't make the interface any heavier.

Believe it or not but I came on github to talk about exactly that :) If you choose week you can use left and right arrows to check other weeks, but if you select last 7 days (not technically a week because it ends on today) you can't easily compare to the 7 days before that etc. That would be my use case, because comparing a monday to the last whole week is a bit limited 😅

**Comment by Razeeman** [comment]
@timkrief Makes sense. Should be also possible to implement.

**Comment by mertbakir** [comment]
This seems to be implemented. We can use <> buttons when custom range selected.

**Comment by Razeeman** [comment]
It is partially implemented, yes. Still need to do this part:

`Another idea would be a more "custom" way to select custom range, for example an option to select months instead of the days.`
